changelog/includeimports.dd

```diff
@@ -1,10 +1,9 @@
-Automatically include imports via -i command line option
+Added the -i command line option to automatically include imports
+
+Added the command line option -i which causes the compiler to treat imported modules as if they were given on the command line.  The option also accepts "module patterns" that include/exclude modules based on their name.  For example, the following will include all modules whose names start with `foo`, except for those that start with `foo.bar`:
+
+$(CONSOLE dmd -i=foo -i=-foo.bar)
 
-Added the command line option -i which causes the compiler to treat imported modules as if they were given on the command line.  The option also accepts "module patterns" that include/exclude modules based on their name.  For example, the following will include all modules whose names start with "foo", except for those that start with "foo.bar":
----
-dmd -i=foo,-foo.bar
----
 The option -i by itself is equivalent to:
----
-dmd -i=-std,-core,-etc,-object
----
+
+$(CONSOLE dmd -i=-std,-core,-etc,-object)

```

Comment:
I thought this syntax was removed with this PR.

Reply:
indeed, unless https://github.com/dlang/dmd/pull/7863 gets merged before we release

Reply:
Shoot, missed that one

Reply:
@ZombineDev do you want to make a PR for that one or shall I?  I think at this point it's gotta be merged to the `stable` branch